### DISCUSSION TITLE
Can't download videos from justforfans

### DISCUSSION THREAD

**Discussion by flummelaa** [AAndyProgram/SCrawler#201] [Q&A]
As written above i can only download pictures from the subscribed site, but no video will be downloaded. Using the newest scrawler version. Please help

Greetings

**Comment by AAndyProgram** [comment]
**Join our Discord server**: https://discord.gg/uFNUXvFFmg

![image](https://github.com/AAndyProgram/SCrawler/assets/88590076/cfa5ca59-3667-42a8-815f-896b4bb51eee)

**Reply by flummelaa** [reply]
Tried to combine the segments, but all i got is that corrupted video
<details>
<summary>Screenshot</summary>
<img src="https://github.com/AAndyProgram/SCrawler/assets/136259324/ae96dd3f-adee-42e1-94fa-4ae120b80564" />
</details>
when i try to decode single frames i get:

```batch
[h264 @ 000001c8eeca0840] top block unavailable for requested intra mode -1
[h264 @ 000001c8eef70ec0] error while decoding MB 10 0, bytestream 23119
[vist#0:0/h264 @ 000001c8ee7e5540] [dec:h264 @ 000001c8ee812d80] corrupt decoded frame
```

and a grey picture

**Reply by AAndyProgram** [reply]
The same here! 😞

**Reply by flummelaa** [reply]
Might be some encryption problem. But i don't now how to pass :|

**Reply by flummelaa** [reply]
so for now, i found out, that the decryption is a problem. Do you know a way to decrypt it? I got a private key.pem and client id.bin from another project. Is there a way to provide it to the crawler?

**Reply by bertvandepoel** [reply]
It's encrypted using ClearKey. I don't know if SCrawler has support for this?

**Reply by AAndyProgram** [reply]
> It's encrypted using ClearKey. I don't know if SCrawler has support for this?

https://github.com/AAndyProgram/SCrawler/blob/main/FAQ.md#sites-questions

**Reply by bertvandepoel** [reply]
Thanks for the link. I had been looking for a GitHub issue. I didn't know you were tracking issues on Discord.

**Reply by AAndyProgram** [reply]
Most users get help on our Discord server.
